src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -757,208 +758,215 @@ export const ShippingRoute: FC<ShippingProps> = props => {
   }
 
   return (
-    <Box data-test="orderShipping">
-      <OrderRouteContainer
-        order={order}
-        currentStep="Shipping"
-        steps={isOffer ? offerFlowSteps : buyNowFlowSteps}
-        content={
-          <Flex
-            flexDirection="column"
-            style={isCommittingMutation ? { pointerEvents: "none" } : {}}
-          >
-            {/* TODO: Make RadioGroup generic for the allowed values,
+    <AnalyticsContext.Provider
+      value={{
+        contextPageOwnerId: order.internalID,
+        contextPageOwnerType: OwnerType.ordersShipping,
+      }}
+    >
```

Comment:
Need to double check that this is still necessary after #12776 which just merged.

Reply:
This is no longer valid as of that PR, as we've changed the API

Reply:
Thanks. I think I updated for that - Let me know if i missed something

Reply:
Looks good!